ci-operator/jobs/openshift/release/openshift-release-master-periodics.yaml

```diff
@@ -30947,7 +30947,7 @@ periodics:
       secret:
         secretName: result-aggregator
 - agent: kubernetes
-  cluster: build09
+  cluster: build04
```

Comment:
sorry but what's all this about changing clouds and clusters?

Reply:
The jobs file is generated from config changes via `make jobs`.  I just trust the computers...